src/app/components/Figure/Caption/index.jsx

```diff
@@ -1,11 +1,19 @@
 import styled from 'styled-components';
 import { FF_NEWS_SANS_REG, C_STORM } from '../../../lib/constants/styles';
+import mediaQuery from '../../../helpers/mediaQueries';
 
 const Caption = styled.figcaption`
   background-color: #d5d0cd;
   color: ${C_STORM};
   font-family: ${FF_NEWS_SANS_REG};
   padding: 8px;
+
+  // Font styling below is a subset of BBC GEL Typography "Long Primer"
+  font-size: 0.9375em;
```

Comment:
This is used in multiple places and could be refactored to be more reusable.

Reply:
@jtart  Are you suggesting to pull out the single lines `font-size: 0.9375em;` or the whole group? e.g.:
```
font-size: 0.9375em;
line-height: 1.125em;
```
Just asking for clarification, since I'm not sure the first way would be useful. (What would you even call it?)
The second could fit under a value of the form `longPrimerGroupAB`, but this seems needlessly complicated.

Reply:
I'm happy to not make the change in this PR as it is minimal duplication, but once our CSS gets more complex we might want to think about it

Reply:
OK, I'm still not clear about at what level you want to abstract things out. e.g. for each line of CSS that's duplicated, or the whole typography group.  It would be useful to know, for future PRs. :)